data/campaigns/Winds_of_Fate/units/elementals/Storm_Daemon.cfg

```diff
@@ -0,0 +1,172 @@
+#textdomain wesnoth-wof
+
+[unit_type]
+    id=Storm Daemon
+    name= _ "Storm Daemon"
+    race=elementals
+    image="units/elementals/storm-1.png"
+    description= _ "There are many tales of the strange storms which roam the Great Ocean and batter its shores. A number tell of twisting thunderheads moving against the wind in pursuit of ill–fated ships. These immensely powerful entities are most commonly known as Storm Daemons. Among learned magi, it is understood they are a kind of wisp, like those sighted drifting over the outlying marshes or glimmering faintly in the deep woods. Whatever else may be known of Storm Daemons is kept a secret of the elves. And perhaps also the silver order of magi, which has long studied such phenomena."
+
+    # Recruiting
+    usage=mixed fighter
+    cost=100
+
+    # Leveling
+    level=5
+    experience=300
+    advances_to=null
+    {AMLA_DEFAULT}
+
+    # Abilities
+    [abilities]
+        {ABILITY_REGENERATES}
+        {ABILITY_SKIRMISHER}
+    [/abilities]
+    {TRAIT_ELEMENTAL}
+
+    # Movement
+    movement=4
+    [movement_costs]
+        deep_water=1
+        shallow_water=1
+        reef=1
+        swamp_water=1
+        flat=1
+        sand=1
+        forest=1
+        hills=1
+        mountains=1
+        village=1
+        castle=1
+        cave=2
+        fungus=1
+        frozen=1
+        unwalkable=1
+    [/movement_costs]
+
+    # Defense
+    hitpoints=120
+    [defense]
+        deep_water=60
+        shallow_water=60
+        reef=60
+        swamp_water=60
+        flat=60
+        sand=60
+        forest=60
+        hills=60
+        mountains=60
+        village=60
+        castle=60
+        cave=70
+        fungus=60
+        frozen=60
+        unwalkable=60
+    [/defense]
+    [resistance]
+        blade=50
+        pierce=50
+        impact=50
+        fire=100
+        cold=100
+        arcane=130
+    [/resistance]
+
+    # Offense
+    alignment=neutral
+    [attack]
+        name=whirlwind
+        description=_"whirlwind"
+        icon=attacks/whirlwind.png
+        range=melee
+        type=impact
+        damage=40
+        number=1
+    [/attack]
+    [attack]
+        name=squall
+        description=_"squall"
+        icon=attacks/squall.png
+        range=ranged
+        type=cold
+        damage=24
+        number=1
+        [specials]
+            {WEAPON_SPECIAL_MAGICAL}
+            {WEAPON_SPECIAL_BLAST (squall) 24 (cold)}
+        [/specials]
+    [/attack]
+    [attack]
+        name=lightning
+        description=_"lightning"
+        icon=attacks/lightning.png
+        range=ranged
+        type=arcane
```

Comment:
this should be either `fire` or `electric`
if you want to make it `arcane` change the attack name.

Reply:
Does `electric` exist…? I'd say `fire` is correct here.

Reply:
The actual BfW UI icon for arcane is a [lightning bolt](https://github.com/wesnoth/wesnoth/blob/master/images/icons/profiles/arcane.png). Plus the arcane (and cold) attacks of the Necromancer line look and sound distinctly electrical.

The other existing ranged arcane attacks are 'lightbeam', 'shadow wave' and 'faerie fire'. So perhaps 'arcane' means 'energy' in modern terminology.

The only two core instances of lightning delivering a fire attack are the incredibly under powered elder mage (Delfador) and the recently added and unused Jinni unit. Plus the storm trident and rod of justice in some campaigns. All of these may be minor enough that we could switch them over to arcane damage to match its lightning bolt symbol?